R/summariseOmopTable.R

```diff
@@ -228,7 +269,10 @@ addVariables <- function(x,
             by = "person_id"
           ) |>
           dplyr::filter(
-            .data$date >= .data$obs_start & .data$date <= .data$obs_end
+            .data$start_date >= .data$obs_start &
+              .data$start_date <= .data$obs_end &
+              .data$end_date >= .data$obs_start &
+              .data$end_date <= .data$obs_end
```

Comment:
I thought we would sumarise in observation start_date and in observation end date separately